static/scripts/topicEdit.js

```diff
@@ -660,7 +701,8 @@ class TopicGeoGebra extends TopicBlock {
                         title={$t('topic.topicEdit.label.youllFindTheIdOn')}><i className="fa fa-info-circle" /></a>
```

Comment:
We can have a button instead of an anchor element

Reply:
I think that would be irritating here, as the element has no functionality on click/ is no button
![anchor](https://user-images.githubusercontent.com/45626086/94259486-d0824c00-ff2e-11ea-9f70-39e68c63c38c.png)

Reply:
So why do we have it as a link? maybe a simple span should be okay?

Reply:
done